bom/pom.xml

```diff
@@ -901,7 +891,6 @@
                 <groupId>org.apache.kafka</groupId>
                 <artifactId>kafka_2.12</artifactId>
                 <version>${kafka2.version}</version>
-                <scope>test</scope>
```

Comment:
Same question about the scope here. You don't set it in other parts of the code.

Reply:
Because the `test` scope was already used.